src/System.Private.CoreLib/shared/System/MemoryExtensions.Fast.cs

```diff
@@ -220,15 +220,12 @@ public static int LastIndexOf(this ReadOnlySpan<char> span, ReadOnlySpan<char> v
         /// <param name="destination">The destination span which contains the transformed characters.</param>
         /// <param name="culture">An object that supplies culture-specific casing rules.</param>
         /// <remarks>If the source and destinations overlap, this method behaves as if the original values are in
-        /// a temporary location before the destination is overwritten.</remarks>
+        /// a temporary location before the destination is overwritten. If <paramref name="culture"/> is null, <see cref="System.Globalization.CultureInfo.CurrentCulture"/> will be used.</remarks>
```

Comment:
I was about to suggest updating the dotnet/api-docs repo with this change as well, but looks like we don't yet have that populated :)

https://docs.microsoft.com/en-us/dotnet/api/system.memoryextensions.tolower?view=netcore-3.0
https://github.com/dotnet/dotnet-api-docs/blob/db066217ac101dd23714cd9da296fc550339db09/xml/System/MemoryExtensions.xml#L3022

cc @mairaw, @carlossanlop